fdbserver/WaitFailure.actor.cpp

```diff
@@ -42,7 +42,7 @@ ACTOR Future<Void> waitFailureClient(RequestStream<ReplyPromise<Void>> waitFailu
 		try {
 			state double start = now();
 			ErrorOr<Void> x = wait(waitFailure.getReplyUnlessFailedFor(ReplyPromise<Void>(), reactionTime, reactionSlope, taskID));
-			if (!x.present()) return Void();
+			if (!x.present()) { return Void(); } 
```

Comment:
I made a change here in 6.3 branch, which will cause merge conflict when merging 6.3. Can you revert this line and merge 6.3 instead?

Reply:
Looks like you already merged 6.3 and this is a conflict now.